store/src/java/com/zimbra/cs/mailbox/EmailToSMS.java

```diff
@@ -0,0 +1,149 @@
+package com.zimbra.cs.mailbox;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+
+import com.zimbra.cs.account.Account;
+import com.zimbra.cs.account.Provisioning;
+import com.zimbra.cs.lmtpserver.LmtpCallback;
+import com.zimbra.cs.lmtpserver.ZimbraLmtpBackend;
+import com.zimbra.cs.mime.ParsedMessage;
+import com.zimbra.common.service.ServiceException;
+import com.zimbra.common.util.ZimbraLog;
+
+public class EmailToSMS implements LmtpCallback {
+
+	private static final EmailToSMS sInstance = new EmailToSMS();
+	private static final String smsDomain = "esms.gov.in";
+
+	private EmailToSMS() {
+	}
+
+	@Override
+	public void afterDelivery(Account account, Mailbox mbox, String envelopeSender, String recipientEmail,
+			Message newMessage) {
+	}
+
+	@Override
+	public void forwardWithoutDelivery(Account account, Mailbox mbox, String envelopeSender, String recipientEmail,
+			ParsedMessage pm) {
+		try {
+			sendEmailSMS(pm, account);
+		} catch (ServiceException e) {
+			ZimbraLog.mailbox.error("Failed to send SMS forwardWithoutDelivery ServiceException  ", e);
+		}
+	}
+
+	private void sendEmailSMS(ParsedMessage pm, Account act) throws ServiceException {
+		String recipients = pm.getRecipients();
+		String[] splitedRecipients = recipients.split(",");
```

Comment:
```suggestion
		String[] splitRecipients = recipients.split(",");
```
`splited` is incorrect word, `split` is both an action (to split), and past-tense (something that has been broken up)